data/hai outfits.txt

```diff
@@ -114,7 +114,7 @@ outfit "Railgun"
 		"hit force" 44
 		"stream"
 		"submunition" "rslug" 1
-	description "The Hai created this weapon for their Flea drones, providing mobile long range support for their close quarters Pond Strider carrier. This particular incarnation of a rail weapon imparts a massive static electric charge to the projectile's flux capacitor, and motivates the slug with electric repulsion. Once in flight, the projectile's charge will bend trajectory very slightly, and allows it to damage the hull a little bit through shields."
+	description "The Hai created this miniaturized incarnation of the classic railgun for their Flea drones, providing mobile long range support for the close quarters Pond Strider carrier."
```

Comment:
when i googled "rail gun", it spit this out:
A railgun is an electromagnetic projectile launcher based on similar principles to the homopolar motor.

rail guns design can be as diverse as our own modern gunpowder based projectile weapons: from muskets to handguns, through rifles and shotguns, rotary auto-cannons, and heavy artillery and mortars: we coloquially call them all "guns", even though the term isn't accurate for most of them.when i googled "rail gun", it spit this out:
A railgun is an electromagnetic projectile launcher based on similar principles to the homopolar motor.

rail guns design can be as diverse as our own modern gunpowder based projectile weapons: from muskets to handguns, through rifles and shotguns, rotary auto-cannons, and heavy artillery and mortars: we coloquially call them all "guns", even though the term isn't accurate for most of them.